### DISCUSSION TITLE
Scrollable table leaves residual border

### DISCUSSION THREAD

**Discussion by ryan-reichenberg** [tailwindlabs/tailwindcss#3200] [Help]
### Describe the problem:

I'm not sure if this is a CSS/tailwind bug or if it is something I'm doing wrong, but I observed this behavior while using Tailwind;
When creating a scrollable table, the container seems to be leaving a border behind at the point where the container ends before scrolling
This happens on Chrome and Firefox.

### Link to a minimal reproduction:

https://play.tailwindcss.com/Af6gaRCIk6?size=326x720

**Comment by adamwathan** [comment]
Can you provide a screenshot/video that demonstrates exactly what you mean? Looked on my phone and couldn’t really figure out what you are trying to point out.

Either way it’s definitely not an actual bug in Tailwind, whatever it is would also happen if you just wrote the same CSS by hand. If we provided a “responsive-table” class or something it would be a bug but Tailwind is just setting individual CSS properties so the browser is just doing whatever it would do with vanilla CSS too so going to transfer to discussions 👍🏻

**Reply by ryan-reichenberg** [reply]
Thanks for the reply!  I would have come here in the first place but I didn't know it existed! I could have explained it better. If you scroll across on the table, there is a very subtle gray line  intersecting the text. 

Here is an example:
![image](https://user-images.githubusercontent.com/26314009/103171809-34584000-48a3-11eb-9b70-dc38add4dc92.png)

![2020-12-28 00 36 25](https://user-images.githubusercontent.com/26314009/103172077-f3612b00-48a4-11eb-99fa-29dfac262b09.gif)
